daft/udf.py

```diff
@@ -30,11 +30,32 @@ class PartialUDF:
     bound_args: inspect.BoundArguments
 
     def expressions(self) -> dict[str, Expression]:
-        return {key: val for key, val in self.bound_args.arguments.items() if isinstance(val, Expression)}
+        parsed_expressions = {}
+        for key, val in self.bound_args.arguments.items():
+            if isinstance(val, Expression):
+                parsed_expressions[key] = val
+            elif isinstance(val, tuple) and all(isinstance(x, Expression) for x in val):
```

Comment:
I think we can relax `all(isinstance(x, Expression) for x in val)` check. we can instead perform the `isinstance(val, Expression)` check inside the inner loop. 
An example would be something like
```
add_cols_elementwise(col('x'), 4, col("y"))
```

Reply:
makes sense, updated